lodash.js

```diff
@@ -132,7 +132,7 @@
   var reIsHostCtor = /^\[object .+?Constructor\]$/;
 
   /** Used to detect unsigned integer values. */
-  var reIsUint = /^\d+$/;
+  var reIsUint = /^(?:\d|[1-9]\d+)$/
```

Comment:
Since you're contributing a bit more I'm going to get a bit more nit picky (I'll easy into it).
Can you run `npm run style` and see if it warns of any style issues.